src/azure-cli/azure/cli/command_modules/backup/commands.py

```diff
@@ -34,45 +38,47 @@ def load_command_table(self, _):
         g.custom_command('backup-properties set', 'set_backup_properties', client_factory=backup_storage_configs_cf)
         g.custom_command('delete', 'delete_vault', confirmation=True)
 
-    with self.command_group('backup container', backup_custom, client_factory=backup_protection_containers_cf) as g:
-        g.show_command('show', 'show_container')
-        g.command('list', 'list_containers', table_transformer=transform_container_list)
+    with self.command_group('backup container', backup_custom_base, client_factory=protection_containers_cf) as g:
```

Comment:
As you shown above, backup_custom_base is custom command type. You need to use custom_command_type kwargs and use custom_command to register your custom command.  [document](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands) and [samples](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/vm/commands.py#L181).